chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/TestingLabController.java

```diff
@@ -57,9 +56,8 @@
 
     @ApiOperation(value = "List all testing labs (ATLs).",
             notes = "Setting the 'editable' parameter to true will return all ATLs that the logged in user has edit "
-                    + "permissions on.  Setting 'showDeleted' to true will include even those ATLs that have been "
-                    + "deleted. The logged in user must have ROLE_ADMIN to see deleted ATLs. The default behavior of "
-                    + "this service is to list all of the ATLs in the system that are not deleted.")
+                    + "permissions on.  Security Restrictions: ROLE_ADMIN or ROLE_ONC can see all ATLs.  ROLE_ATL "
```

Comment:
This phrasing implies that anon cannot call this method, but this is called to populate the /overview page, so I know it works there

Reply:
Addressed